server/src/serializers/BoardGameSerializer.js

```diff
@@ -0,0 +1,26 @@
+import { Review } from "../models/index.js"
+import ReviewSerializer from "./ReviewSerializer.js"
+
+class BoardGameSerializer {
+  static async getSummary(boardGame) {
+
+    const allowedAttributes = ["id", "name", "minPlayers", "maxPlayers", "estimatedPlayTime", "description"]
+
+    let serializedBoardGame = {}
+
+    for (const attribute of allowedAttributes) {
+      serializedBoardGame[attribute] = boardGame[attribute]
+    }
+
+    const boardGameId = boardGame.id
+    const reviewsData = await Review.query().where({boardGameId})
```

Comment:
here I would instead encourage using `$relatedQuery` for the `serializedReview`, that way we don't need to import the Review model
```js
const reviewData = serializedBoardGame.$relatedQuery("reviews")
```